ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/vmgenv1/provision/cucushift-installer-rehearse-azure-ipi-vmgenv1-provision-chain.yaml

```diff
@@ -0,0 +1,7 @@
+chain:
+  as: cucushift-installer-rehearse-azure-ipi-vmgenv1-provision
+  steps:
+  - chain: ipi-conf-azure-vmgenv1
+  - chain: ipi-install 
+  documentation: |-
+    Create an IPI cluster which vm generate is v1 on AZURE for QE e2e tests.
```

Comment:
Pls add some description to say this testing is meaningful only for 4.11, later versions